src/System.Diagnostics.TraceSource/tests/DefaultTraceListenerClassTests.cs

```diff
@@ -27,6 +28,11 @@ public override void Write(string message)
             {
                 _writer.Write(message);
             }
+
+            public override void WriteLine(string message)
+            {
+                _writer.WriteLine(message);
+            }
```

Comment:
Because `DefaultTraceListener.Fail(string)` in desktop calls `WriteLine` instead of `Write` which we call in Core